src/Microsoft.ServiceFabric.Actors/Runtime/ActorConcurrencyLock.cs

```diff
@@ -174,18 +174,18 @@ internal int Test_CurrentCount
                         timeout));
             }
 
-            // the caller has the turn lock 
+            // The caller has the turn lock. 
             try
             {
-                // check  if the owner is dirty
+                // Checks  if the owner is dirty.
                 if (this.owner.IsDirty && handler != null)
                 {
-                    // call dirty state handler to handle it
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.